packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -71,6 +71,7 @@ interface ToolConfiguration {
       opacity: number;
       handleRadius: number;
     };
+    jumpThreshold: number;
```

Comment:
could you please add the documentation for the `jumpThreshold` ? I'm not sure what it does